pkg/apply/event/event.go

```diff
@@ -154,4 +159,5 @@ type DeleteEvent struct {
 	Type      DeleteEventType
 	Operation DeleteEventOperation
 	Object    runtime.Object
+	Error     error
```

Comment:
I would assume the `DeleteEvent` should look identical to the `PruneEvent`. Why no `ObjectMeta` here?

Reply:
My bad, missed that part. Will add it.

Reply:
Updated this